test/python/scheduler/test_basic_scheduler.py

```diff
@@ -169,10 +170,17 @@ def test_measure_combined(self):
             (28, self.cmd_def.get('cx', [0, 1])),
             (50, self.cmd_def.get('measure', [0, 1])),
             (60, self.cmd_def.get('measure', [0, 1])))
-        for actual, expected in zip(sched.instructions, expected.instructions):
```

Comment:
Keep these lines. Rather than try to change the test itself, first modify `expected` before this line. Modify it to match what your expectations are.